src/usb.c

```diff
@@ -604,7 +604,7 @@ s8 usb_64drive_wait()
         #endif
         
         // Took too long, abort
-        if((timeout++) > 1000000)
+        if((timeout++) > 10000)

```

Comment:
We probably need to PR these to the original source if we haven't already. Eve though this is a small lib, we may get out of sync very quickly

Reply:
Done & merged already: https://github.com/buu342/N64-UNFLoader/pull/55